### PR TITLE
DP-2368 #Pushing local changes

### PR DESCRIPTION
**Pull request by niharikay**

### CODE REVIEW

File: contrib/docker/install-dependency.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+rpm -ivh mysql-community-release-el7-5.noarch.rpm
+yum update
+yum install -y mysql-server
+systemctl start mysqld
+yum install -y git
+mkdir /home/superset
+cd /home/superset
+git clone https://niharikay@github.com/analyticsMD/incubator-superset.git
+pip install docker-compose
+docker login -uqventusci -p3FvK7kN?]Lie4
```
Comment (dfudenb) [inline contrib/docker/install-dependency.sh:16]: rm

File: contrib/docker/install-dependency.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+yum install -y wget
+wget http://repo.mysql.com/mysql-community-release-el7-5.noarch.rpm
+rpm -ivh mysql-community-release-el7-5.noarch.rpm
+yum update
+yum install -y mysql-server
+systemctl start mysqld
+yum install -y git
+mkdir /home/superset
+cd /home/superset
+git clone https://niharikay@github.com/analyticsMD/incubator-superset.git
```
Comment (dfudenb) [inline contrib/docker/install-dependency.sh:14]: what is stored here? -- should make user independent --

**Review by dfudenb — CHANGES_REQUESTED** [review]
rm pwds

**Comment by niharikay** [comment]
This PR is no longer required. All recent changes are moved to branch: superset-0.31.3 and the new container is pushed to Dockerhub(qventus/superset-0.31.4).

**Pull request closed without merging**